src/app/pages/network/components/open-vpn-server-config/open-vpn-server-config.component.html

```diff
@@ -10,19 +10,24 @@
               [label]="'Server Certificate' | translate"
               [required]="true"
               [options]="serverCertificates$"
-              (linkClicked)="certificatesLinkClicked()"
-              [linkText]="'Certificates' | translate"
-              [tooltip]="tooltips.server_certificate | translate"
+              class="linked-input"
             ></ix-select>
+            <a class="link" (click)="certificatesLinkClicked()">
+              {{ 'Certificates' | translate }}<mat-icon>open_in_new</mat-icon>
+            </a>
```

Comment:
I think it's better to use `routerLink`
```
<div class="link">
    <a [routerLink]="['/', 'credentials', 'certificates']">
      {{ 'Certificates' | translate }}<mat-icon>open_in_new</mat-icon>
    </a>
  </div>

Reply:
@AlexKarpov98 I used a method because sometimes, you have to close the slide in panel or do something else while redirecting. Using routerLink doesn't let you do that. So the back page gets redirected but the slide in panel remains open in the front.

Reply:
![routerlink issue](https://user-images.githubusercontent.com/24204846/161075685-a034d8a7-4391-484e-abc4-a42ab0b5eff7.gif)